SidebarPanel.js

```diff
@@ -216,7 +216,10 @@ class SidebarPanel {
           if(imageUrl.startsWith("data:")){
             alert("You cannot use urls starting with data:");
```

Comment:
Looking at this one again, I think we need to check for `data:` on all of them. Either that, or we need to update the `parse_img` function to check for that. I don't expect people to try sneaking any in, but we better guard against it just in case.